### ISSUE TITLE
Due to bad actors I have been forced to shutdown the 'Issues' section until further notice

### ISSUE THREAD

**Issue by iamadamdev** [iamadamdev/bypass-paywalls-chrome#538]
A certain user is repeatedly making new accounts everyday and trying to spam my repo in a desperate attempt to attract more users.

**Comment by hongkong664** [comment]
Indeed, @magnolia1234 repo supports more sites, has no Google Analytics and you can add your own custom sites.
Only 'understandable' you're still copying his fixes (for months now) ;-)
